ac/ac-autocode/src/ActivityRunner.jsx

```diff
@@ -1,30 +1,91 @@
 // @flow
 
-import React from 'react';
+import React, { Component } from 'react';
 import type { ActivityRunnerT } from 'frog-utils';
 
-const ActivityRunner = ({ activityData }: ActivityRunnerT) => {
-  const submitCode = () => {};
+export default class ActivityRunner extends Component {
+  state: {
+    inputCode: string,
+    outputFeed: string
+  };
 
-  return (
-    <div>
-      <h1>{activityData.config.title || 'Coding exercise'}</h1>
+  constructor(props: ActivityRunnerT) {
+    super(props);
+
+    this.state = {
+      inputCode: 'print "Hello World"',
+      outputFeed: "You'll see the output of your code here"
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.runit = this.runit.bind(this);
+  }
+
+  componentDidMount() {
+    const script = document.createElement('script');
```

Comment:
Using Skulpt require to execute the two following scripts, which create the object 'Sk'.
The current solution calls the scripts from the Skulpt distribution site and has them included on the page.
The object will then be referred by window.Sk